src/validator.directive.js

```diff
@@ -288,7 +289,7 @@
          */
         if (attrs.validMethod === 'blur') {
           element.bind('blur', function() {
-            var value = ctrl.$viewValue;
+            var value = scope.$eval(attrs.ngModel);
```

Comment:
It is the same but since we have ng-model defined as our directive attr I
think it is better to use that rather than controller view value. Then what
would be the usage of ngModel?
On Nov 29, 2015 9:51 PM, "Huei Tan" <notifications@github.com> wrote:

> In src/validator.directive.js
> <https://github.com/huei90/angular-validation/pull/162#discussion_r46110404>
> :
>
> > @@ -288,7 +289,7 @@
> >           */
> >          if (attrs.validMethod === 'blur') {
> >            element.bind('blur', function() {
> > -            var value = ctrl.$viewValue;
> > +            var value = scope.$eval(attrs.ngModel);
>
> What's the difference between ctrl.$viewValue and
> scope.$eval(attrs.ngModel) in our model.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/huei90/angular-validation/pull/162/files#r46110404>.
>